### PR TITLE
feat: access bandeau 

### PR DESCRIPTION
**Pull request by akornne**

### CODE REVIEW

**Review by bayram-69 — APPROVED** [review]

File: frontend/src/main.jsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ import RecipesPage from "./pages/RecipesPage";
 import UserPage from "./pages/UserPage";
 import UserInformation from "./components/UserInformation";
 import AdminPage from "./pages/AdminPage";
+import Registration from "./pages/RegistrationPage";
+import Login from "./pages/LoginPage";
```
Comment (PierreA44) [inline frontend/src/main.jsx:15]: name : LoginPage instead Login

**Review by ioayoub — APPROVED** [review]

File: frontend/src/main.jsx [hunk 2]
```diff
@@ -70,6 +72,14 @@ const router = createBrowserRouter([
           },
         ],
       },
+      {
+        path: "/registration",
+        element: <Registration />,
+      },
+      {
+        path: "/Login",
+        element: <Login />,
```
Comment (PierreA44) [inline frontend/src/main.jsx:81]: so rename to LoginPage

File: frontend/src/pages/LoginPage.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import Login from "../components/Login";
+
+export default function LoginForm() {
```
Comment (PierreA44) [inline frontend/src/pages/LoginPage.jsx:3]: function LoginForm in LoginPage file ?

change please

File: frontend/src/main.jsx [hunk 4]
```diff
@@ -70,6 +72,14 @@ const router = createBrowserRouter([
           },
         ],
       },
+      {
+        path: "/registration",
+        element: <Registration />,
+      },
+      {
+        path: "/Login",
```
Comment (PierreA44) [inline frontend/src/main.jsx:80]: write the path in lowercase please

File: frontend/src/components/RegistrationForm.jsx [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
   } = useForm();
 
-  const onSubmit = (data) => {
-    axios.post(`${import.meta.env.VITE_BACKEND_URL}/api/user`, data);
+  const onSubmit = async (data) => {
+    await axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/api/user`, data)
+      .then((res) => console.info(res.data));
+    localStorage.setItem("accessVisible", true);
+    navigate("/Login");
```
Comment (PierreA44) [inline frontend/src/components/RegistrationForm.jsx:20]: path in lowercase please

File: frontend/src/components/Navbar.jsx [hunk 6]
```diff
@@ -9,6 +9,15 @@ function Navbar() {
         <img className="h-28 rounded-full" src={logo} alt="logo" />
       </NavLink>
       <div className="text-beige m-4 flex gap-3">
+        <NavLink className="hover:text-green text-xl h-fit my-auto" to="/Login">
+          Connexion
+        </NavLink>{" "}
+        <NavLink
+          className="hover:text-green text-xl h-fit my-auto"
+          to="/Registration"
```
Comment (PierreA44) [inline frontend/src/components/Navbar.jsx:17]: path in lowercase please

File: frontend/src/components/Access.jsx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+          <p className="text-center leading-6 text-gray-900 text-2xl">
+            <strong className="font-semibold">Eating nam nam 2024</strong>
+            <svg
+              viewBox="0 0 2 2"
+              className="mx-2 inline h-0.5 w-0.5 fill-current"
+              aria-hidden="true"
+            >
+              <circle cx={1} cy={1} r={1} />
+            </svg>
+            Rejoigner nous pour plus de recettes de partage et de bonne humeur.
```
Comment (QuentinVerwaerde) [inline frontend/src/components/Access.jsx:28]: Can you fix the french errors please ? "rejoignez" and add "," between "recettes" and "de"

**Review by PierreA44 — CHANGES_REQUESTED** [review]
checks my comments

**Review by Mewlyscin — APPROVED** [review]

File: frontend/src/components/Access.jsx [hunk 8]
```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { NavLink, useNavigate } from "react-router-dom";
+import RegistrationForm from "./RegistrationForm";
+
+export default function Access({ setAccessVisible }) {
+  const navigate = useNavigate();
+  const [isVisible, setIsVisible] = useState(false);
```
Comment (QuentinVerwaerde) [inline frontend/src/components/Access.jsx:8]: Please add a comment on what this state is used for
Same comment for "option"

**Review by QuentinVerwaerde — CHANGES_REQUESTED** [review]

**Review by QuentinVerwaerde — APPROVED** [review]

**Review by PierreA44 — APPROVED** [review]

**Pull request merged**